operatorconfig/driverconfig/unity/v2.11.1/csidriver.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+    name: csi-unity.dellemc.com
+spec:
+    attachRequired: true
+    podInfoOnMount: true
+    storageCapacity: true
+    volumeLifecycleModes:
+      - Persistent
+      - Ephemeral
+    fsGroupPolicy: ReadWriteOnceWithFSType
```

Comment:
Add a new line at the end

Reply:
Is the ask here to go through every YAML file and ensure that they have a newline at the end? That could take hours and would be easily resolved by a linter, but we're pulling the linter changes out because it was almost fifty thousand additional lines of change.